### PR TITLE
Update terms of service

### PR DESCRIPTION
**Pull request by leoasis**
Updating terms of service with the latest version on the website.

### CODE REVIEW

**Review by jutretel — APPROVED** [review]

File: terms_of_service/pl/terms_of_service.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 WARUNKI ŚWIADCZENIA USŁUG 
-Ostatnia modyfikacja: 2024-01-29
+Last Modified: 2024-09-06
```
Comment (mwgamera) [inline terms_of_service/pl/terms_of_service.txt:2]: Is “Last Modified” intentionally left in English in all the language versions?

File: terms_of_service/pl/terms_of_service.txt [hunk 2]
```diff
@@ -31,15 +31,15 @@ W miarę rozwoju i udoskonalania serwisu Tumblr możemy być zmuszeni do wprowad
...
+You may not use the Services, provide any personal information to Tumblr, or otherwise submit personal information through the Services (including, for example, a name, address, telephone number, or email address) if you are under the Minimum Age. The Minimum Age is (i) thirteen (13), (ii) sixteen (16) for users in the European Union or the United Kingdom, or (iii) eighteen (18) for certain features. You may only use the Services if you can form a binding contract with Tumblr and are not legally prohibited from using the Services.
```
Comment (mwgamera) [inline terms_of_service/pl/terms_of_service.txt:34]: Whole untranslated paragraph in English. It seems that it was left in English in other language versions too.

**Comment by leoasis** [comment]
@jutretel @mwgamera seems the translations are not completely done yet, will wait until they're done and update this PR!

**Comment by leoasis** [comment]
I've updated the policy as it seems we have most languages translated now. The only one still using English for the new text is Polish. Will wait for a few days to see if it gets updated.

**Comment by leoasis** [comment]
cc @mwgamera @jutretel this should now have all the translations, so it should be ready to merge.

Notice there's another PR with a newer terms of service version, but I think we should merge this one first as we're still awaiting for translations on the other one, and also so that we have all the versions in history.

**Pull request merged**